firefox_ui_tests/remote/security/test_ssl_disabled_error_page.py

```diff
@@ -0,0 +1,96 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import time
+
+from marionette import By
+from marionette.errors import MarionetteException, TimeoutException
+
+from firefox_ui_harness.testcase import FirefoxTestCase
+
+
+class TestSSLDisabledErrorPage(FirefoxTestCase):
+
+    def setUp(self):
+        FirefoxTestCase.setUp(self)
+
+        self.url = 'https://tlsv1-0.mozqa.com'
+
+        # mozmill: utils.sanitize({ sessions: true }); // necessary? test succeeds without this
+        # also: utils.sanitize used by 3 geolocation tests; better to add to library, maybe prefs?
+        # also: https://dxr.mozilla.org/mozilla-central/source/browser/base/content/sanitize.js#417
+        try:
+            self.marionette.execute_async_script("""
+                Cu.import("resource://gre/modules/Services.jsm");
```

Comment:
This code block clearly has to be moved to a library. I don't know which one so maybe we also put it into a utils.py module under api?